.github/workflows/integration-wf.yml

```diff
@@ -0,0 +1,85 @@
+name: Integration Test Suite WorkFlow
+
+on: 
+  push:
+    branches:
+      - integration_test
```

Comment:
@kunal-bagwe Do you want to change to master here, as in this repo it will run on master branch.

Reply:
yeah, i am testing the flow here, for test purpose its feature branch, we will have master here.